cmd/update.go

```diff
@@ -0,0 +1,94 @@
+package cmd
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/blang/semver"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	githubSlug     = "screwdriver-cd/sd-local"
+	currentVersion = version
+	updateFlag     = false
+)
+
+func canUpdate() (*selfupdate.Release, error) {
+
+	if currentVersion == "dev" {
+		return &selfupdate.Release{}, errors.New("This is a development version and cannot be updated")
+	}
+
+	latest, found, err := selfupdate.DetectLatest(githubSlug)
+	if err != nil {
+		return &selfupdate.Release{}, err
+	}
+	if !found {
+		return &selfupdate.Release{}, errors.New("Repositry Not Found")
+	}
+	v := semver.MustParse(currentVersion)
+
+	if latest.Version.LTE(v) {
+		return &selfupdate.Release{}, errors.New("Current version is latest")
+	}
+
+	return latest, nil
+}
+
+func askUpdateForUser(latestVersion *selfupdate.Release) error {
+	fmt.Print("Do you want to update to", latestVersion.Version.String(), "? [y/N]: ")
+	input, err := bufio.NewReader(os.Stdin).ReadString('\n')
+	if err != nil {
+		return err
+	}
+	input = strings.TrimSuffix(input, "\n")
+	if input == "y" {
+		return nil
+	}
+	return errors.New("Invalid input")
```

Comment:
I think it's a little strange that it causes the error when we input "N".

Reply:
Yes, I'll fix that.